dojo/tools/nowsecure/parser.py

```diff
@@ -0,0 +1,105 @@
+import hashlib
+import json
+from urllib.parse import urlparse
+from dojo.models import Endpoint, Finding
+import re
+
+class NowSecureParser(object):
+    """
+    Importing findings from NowSecure app analysis.
+    """
+
+    def get_scan_types(self):
+        return ["NowSecure Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "NowSecure Scan"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "NowSecure report file can be imported in JSON format (option --json)."
+
+    def get_findings(self, file, test):
+        data = json.load(file)
+
+        dupes = dict()
+        for content in data['data']['auto']['assessment']['report']['findings']:
+            if  content['affected']:
+                if content['check']['issue']:
+                    title = content['check']['issue']['title']
+                    for regulation in content['check']['issue']['regulations']:
+                        if regulation['type'] == "cwe":
+                            cwe = regulation['links'][0]['title']
+                    description = content['check']['issue']['description']
+                    severity = content['check']['issue']['severity'].capitalize()
+                    cvssv3 = content['check']['issue']['cvssVector']
+                    cvssv3_score = content['check']['issue']['cvss']
+                    mitigation = content['check']['issue']['recommendation']
```

Comment:
Please also add fix_available in case you have a mitigation and add it to the docs